cg/apps/lims/api.py

```diff
@@ -420,6 +426,26 @@ def get_sample_rin(self, sample_id: str) -> float | None:
             LOG.warning(f"Sample {sample_id} not found in LIMS: {error}")
         return rin
 
+    def get_sample_dv200(self, sample_id: str) -> float | None:
+        """Return the sample's percentage of RNA fragments greater than 200 nucleotides."""
+        dv200 = None
+        try:
+            sample_artifact: Artifact = Artifact(self, id=f"{sample_id}PA1")
+            dv200: float = sample_artifact.udf.get(PROP2UDF["dv200"])
+        except HTTPError as error:
+            LOG.warning(f"Sample {sample_id} not found in LIMS: {error}")
+        return dv200
+
+    def get_sample_reception_qc_flag(self, sample_id: str) -> ReceptionQCFLag | None:
+        """Return the sample reception QC flag."""
+        qc_flag = None
+        try:
+            sample_artifact: Artifact = Artifact(self, id=f"{sample_id}PA1")
```

Comment:
is this the same for both DNA and RNA? I can see that in the lims the "step" they belong to have different names

Reply:
At least for MIP WGS samples this value is reported in the Artifact object, but I will make sure it's the same for Balsamic as well